guides/v2.0/config-guide/varnish/tshoot-varnish-503.md

```diff
@@ -10,12 +10,17 @@ version: 2.0
 github_link: config-guide/varnish/tshoot-varnish-503.md
 ---
 
-If the length of {% glossarytooltip 0bc9c8bc-de1a-4a06-9c99-a89a29c30645 %}cache{% endglossarytooltip %} tags used by Magento exceed Varnish's default of 8192 characters, you can see HTTP 503 (Backend Fetch Failed) errors in the browser. The errors might display similar to the following:
+If the length of {% glossarytooltip 0bc9c8bc-de1a-4a06-9c99-a89a29c30645 %}cache{% endglossarytooltip %} tags used by Magento exceed Varnish's default (8K), you can see HTTP 503 (Backend Fetch Failed) errors in the browser. The errors might display similar to the following:
 
 	Error 503 Backend fetch failed
 	Backend fetch failed
 
-To resolve this issue, increase the default value of `http_resp_hdr_len` in your Varnish configuration file as follows:
+To resolve this issue, increase the default value of the `http_resp_hdr_len` parameter in your Varnish configuration file. The `http_resp_hdr_len` parameter specifies the max header length _within_ the total default response size (32K).
+
+<div class="bs-callout bs-callout-info" id="info">
```

Comment:
FYI. You don't have to change this. Use this to define your Note block: 

`<div class="bs-callout bs-callout-info" id="info" markdown="1">`

As a result, you don't need the span, p, and code elements (though you'd still need to use the backticks for the parameter)